src/ctap/credential_management.rs

```diff
@@ -155,12 +171,14 @@ fn process_enumerate_rps_get_next_rp(
 fn process_enumerate_credentials_begin(
     persistent_store: &PersistentStore,
     stateful_command_permission: &mut StatefulPermission,
+    pin_protocol_v1: &mut PinProtocolV1,
```

Comment:
Should we define a PinProtocol trait instead, anticipating that in a short term we will implement PinProtocolV2?

Reply:
I plan to, but I have to design that first. Should go into a separate PR.